v3-0006-efi-Do-not-try-to-load-the-BGRT-under-Xen.patch

```diff
@@ -0,0 +1,33 @@
+From 34e562d135f3bcac842202af53d6b87869926431 Mon Sep 17 00:00:00 2001
+Message-Id: <34e562d135f3bcac842202af53d6b87869926431.1669264419.git.demi@invisiblethingslab.com>
+In-Reply-To: <b18879e0329c785d35f2aa2164413bb56419c684.1669264419.git.demi@invisiblethingslab.com>
+References: <b18879e0329c785d35f2aa2164413bb56419c684.1669264419.git.demi@invisiblethingslab.com>
+From: Demi Marie Obenour <demi@invisiblethingslab.com>
+Date: Wed, 23 Nov 2022 23:31:40 -0500
+Subject: [PATCH v3 6/6] efi: Do not try to load the BGRT under Xen
```

Comment:
Please drop this patch. Xen does know about BGRT table and invalidate it right now, but at some point it may avoid clobbering it.